addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -120,19 +120,30 @@
                 <group>
                     <field name="is_public"/>
                 </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="mailing_list_view_form_simplified_footer" model="ir.ui.view">
```

Comment:
Would give it a higher priority (aka not default, as views are chosen by priority ascending).